sdks/python/apache_beam/runners/portability/job_server.py

```diff
@@ -125,10 +124,6 @@ def local_temp_dir(self, **kwargs):
 
 
 class JavaJarJobServer(SubprocessJobServer):
-
-  MAVEN_REPOSITORY = 'https://repo.maven.apache.org/maven2/org/apache/beam'
-  JAR_CACHE = os.path.expanduser("~/.apache_beam/cache")
```

Comment:
Seems like this is used here ?
https://github.com/apache/beam/blob/b91560cc354da471e3de502aad78dd059997a3d0/sdks/python/apache_beam/utils/subprocess_server.py#L198